setup.py

```diff
@@ -13,17 +14,24 @@
 
 
 def read(fname):
-    with open(os.path.join(os.path.dirname(__file__), fname)) as f:
+    with open(Path(__file__).parent / fname) as f:
         result = f.read()
     return result
 
 
+# Requirements not on PyPI can't be installed through `install_requires`.
+# They have to be installed manually or with `pip install -r requirements.txt`.
+requirements = [r for r in read('requirements.txt').splitlines()
+                if not r.startswith('git+https://')]
```

Comment:
Sorry for being late to the party on this PR. Do we need to amend the README to `pip install ...` instead of/in addition to `python setup.py develop`?